mail-filter/mimedefang/metadata.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<!-- maintainer-needed -->
```

Comment:
We won't add this package unless it has a maintainer. If you want to maintain it, add yourself and proxy-maint team. You can find example snippet on our project page.

Reply:
Please see new commit.